fastscape/tests/test_processes_initial.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+import pytest
+
+from fastscape.processes import (BareRockSurface,
+                                 Escarpment,
+                                 FlatSurface,
+                                 NoErosionHistory)
+
+def test_flat_surface():
+    np.random.seed(5)
+
+    grid = (3, 2)
+    elevation = np.random.rand(*grid)
+
+    np.random.seed(5)
+    p = FlatSurface(shape=grid)
+    p.initialize()
+    
+    np.testing.assert_equal(grid, p.shape)
+    np.testing.assert_equal(elevation, p.elevation)
+
+
+@pytest.mark.parametrize('inputs, expected_result', [
```

Comment:
Move all parameters with invariant values in the function body.

nit: `expected_result` -> `expected_elevation`.